template/module/tests/test_abstract_something.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# Copyright <YEAR(S)> <AUTHOR(S)>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import mock
+
+from openerp import models
+from openerp.tests.common import TransactionCase
+
+
+class AbstractSomethingTester(models.Model):
+    """ It provides a real model object to test the abstract with """
+    _name = 'abstract.something.tester'
+    _description = 'Abstract Something Tester'
+    _inherit = 'abstract.something'
+
+
+class TestAbstractSomething(TransactionCase):
```

Comment:
This has to be `SingleTransactionCase` or `SavepointCase` to work as is. Otherwise, the generation methods need to be moved to `setUp`. Maybe make two example classes instead.